fastapi_poe/src/fastapi_poe/base.py

```diff
@@ -181,6 +181,10 @@ def generate_auth_key(api_key: str) -> str:
     print("Generated API key:")
     print(auth_key)
     print("Set this key in the 'API Key' field in the bot creation form.")
+    print(
+        "Note: This key will be replaced by a newly generated key on server "
+        "restart unless you copy the key into the POE_API_KEY variable in main.py"
```

Comment:
This is a bit too specific to the replit, people would run this same code when they use some other mechanism to run fastapi_poe.
```suggestion
        "restart unless you set the key in your code or in an environment variable"
```